src/components/UtilizationBar/UtilizationBar.js

```diff
@@ -28,8 +28,16 @@ const UtilizationBar = props => {
     }
   };
 
+const mainDivClasses = mainDivClasses => {
+  if (mainDivClasses) {
+    return mainDivClasses + " progress";
+  } else {
+    return "progress";
+  }
+};
```

Comment:
Take a look at https://github.com/patternfly/patternfly-react/blob/master/src/components/EmptyState/EmptyState.js#L9 for an example of using `ClassNames` to do this.